modules/ingest-common/src/yamlRestTest/resources/rest-api-spec/test/ingest/70_bulk.yml

```diff
@@ -167,3 +167,43 @@ teardown:
         index: test_index
         id: test_id3
   - match: { _source: {"f1": "v2", "f2": 47, "field1": "value1"}}
+
+---
+"Test bulk API with batch enabled":
```

Comment:
I think you need more tests for different values of `batch_ingestion_option`, and `batch_size`.

- Batch ingestion option standalone without batch size
- Batch size without batch ingestion option
- Batch size of 0, negative value, etc.
- Invalid batch ingestion option
- Some combinations of the above

Reply:
Since batch_ingestion_option was removed, its related tests are not added. Added test for invalid batch size, missing batch_size tests are covered by existing tests.